conf/eslint-all.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * @fileoverview Config to enable all rules.
+ * @author Robert Fletcher
+ */
+
+"use strict";
+
+//------------------------------------------------------------------------------
+// Requirements
+//------------------------------------------------------------------------------
+
+var lodash = require("lodash"),
+    fs = require("fs"),
+    path = require("path");
+
+//------------------------------------------------------------------------------
+// Helpers
+//------------------------------------------------------------------------------
+
+var ruleFiles = fs.readdirSync(path.resolve(__dirname, "../lib/rules"));
+var enabledRules = lodash.reduce(ruleFiles, function(result, filename) {
```

Comment:
Ha, I had some changes locally where I was trying to remove the `lodash` dependency using `Object.keys`. I had forgotten `reduce` was available on arrays.